### PR TITLE
change set_rtss_epi to set_pev_epi

### PR DESCRIPTION
**Pull request by lmhaile**
Quick change per the new malariasimulation version!

### CODE REVIEW

File: R/interventions.R [hunk 1]
```diff
@@ -238,12 +238,13 @@ add_smc <- function(p, interventions){
...
   month <- 365 / 12
   timesteps <- 1 + (interventions$year - p$baseline_year) * 365
 
-  p <- malariasimulation::set_rtss_epi(
+  p <- malariasimulation::set_pev_epi(
     parameters = p,
+    profile = rtss_profile,
     timesteps = timesteps,
     coverages = interventions$rtss_cov,
     age = round(6 * month),
```
Comment (pwinskill) [inline R/interventions.R:250]: There are some pieces missing here, if we look at https://mrc-ide.github.io/malariasimulation/articles/Vaccines.html#simulation-2 then we need to:

- [ ]  additionally specify the `booster_profile`
- [ ] rename the `boosters` argument to `booster_timestep`
- [ ] `seasonal_boosters` is by default set to `FALSE` so we can remove that

**Comment by pwinskill** [comment]
@lmhaile - we need the checks above to be passing before these changes can be brought in.
You can run `devtools::test()` and `devtools::check()` locally to see what errors and warnings are produced. 

Given that we have made modifications to the vaccine implementation, it is likely that the associated test will also have to be corrected, that is in this file: https://github.com/mrc-ide/site/blob/main/tests/testthat/test-rtss.R. You can read here (https://r-pkgs.org/testing-basics.html) for more info on unit tests in packages if it is new to you.

**Review by pwinskill — CHANGES_REQUESTED** [review]

**Comment by pwinskill** [comment]
In addition, it might be nice to add a plot to this PR showing the two site files runs - one with vaccination and one without. That way we can be more cofident that the vaccine implementation if feeding through to the model as expected

**Comment by lmhaile** [comment]
> In addition, it might be nice to add a plot to this PR showing the two site files runs - one with vaccination and one without. That way we can be more cofident that the vaccine implementation if feeding through to the model as expected

Diagnostic of clinical incidence over time (intervention= 100% vaccine coverage for time period, baseline= 0% coverage):
image


Source code:
```
install.packages('Q:/site_0.2.2.tar.gz')

# model and plotting functions
run_model <- function(site) {
  
  # pull parameters for this site
  params <- site::site_parameters(
    interventions = site$interventions,
    demography = site$demography,
    vectors = site$vectors,
    seasonality = site$seasonality,
    eir = site$eir$eir[1],
    overrides = list(human_population = 1000),
    burnin = 5 * 365
  )
  
  model <-  malariasimulation::run_simulation(timesteps = 20 * 365,
                                              parameters = params)
  return(model)
}

format_for_plots <- function (baseline, int) {
  baseline <- get_rates(
    baseline,
    time_divisor = 365,
    baseline_t = 0,
    age_divisor = 1,
    scaler = 0.215,
    treatment_scaler = 0.5,
    baseline_treatment = 0
  )
  
  baseline <- baseline |>
    mutate(identifier = 'baseline')
  
  int <- get_rates(
    int,
    time_divisor = 365,
    baseline_t = 0,
    age_divisor = 1,
    scaler = 0.215,
    treatment_scaler = 0.5,
    baseline_treatment = 0
  )
  
  int <- int |>
    mutate(identifier = 'intervention')
  
  
  dt <- rbind(baseline, int, fill = T)
  dt <- dt |>
    rename(year = t) |>
    mutate(age = as.numeric(age_lower)) |>
    mutate(age = round(age / 365)) |>
    mutate(year = year + 1999) |>
    select(identifier, year, age, clinical)
  
  
  dt <- data.table(dt)
  
  dt <- dt[identifier != TRUE]
  return(dt)
}


# run one model with vaccine coverage of 100% and another with 0% coverage
# example site
site_data <- foresite::MWI
site <- site::single_site(site_data, 5)

# run with vaccine coverage
rtss <- copy(site)
rtss$interventions$rtss_cov <- 1

# run without
no_rtss <- copy(site)
no_rtss$interventions$rtss_cov <- 0


intvn <- run_model(rtss)
bl <- run_model(no_rtss)

dt <- format_for_plots(baseline = bl, int = intvn)

# plot
p <-
  ggplot(
    data = dt,
    mapping = aes(
      x = year,
      y = clinical,
      color = identifier,
      fill = identifier
    )
  ) +
  geom_point(alpha = 0.5)  +
  facet_wrap_paginate( ~ round(age), scales = 'free') +
  labs(
    x = 'Time (in years)',
    y = 'Clinical incidence rate',
    title = paste0('Clinical incidence rate over time'),
    color = 'Model Run',
    fill = 'Model Run'
  ) +
  theme_light(base_family = 'Arial') +
  scale_color_manual(values = wes_palette('Royal2', n = 2)) +
  scale_x_continuous(guide = guide_axis(n.dodge = 2)) +
  scale_fill_manual(values = wes_palette('Royal2', n = 2))


print(p)
```

**Pull request closed without merging**